pytest_bdd/feature.py

```diff
@@ -141,6 +142,18 @@ def get_tags(line):
     return set((tag.lstrip("@") for tag in line.strip().split(" @") if len(tag) > 1))
 
 
+def get_params(line):
```

Comment:
I think that "split_line" would be a better name for this, since we are not getting parameters from a line, but just splitting it with handling of escape sequences.

Reply:
That makes a lot more sense, yes. Also removed the `if not line` check I had put in there, which I think was unnecessary.